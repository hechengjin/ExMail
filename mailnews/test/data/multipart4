Content-Type: multipart/mixed; boundary=boundary

--boundary

This has no headers, so should be recognized as plain text.

--boundary--
