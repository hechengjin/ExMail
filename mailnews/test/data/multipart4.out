This has no headers, so should be recognized as plain text.